ticketutil/ticket.py

```diff
@@ -67,19 +105,21 @@ def _create_requests_session(self):
             r.raise_for_status()
             logging.debug("Create requests session: Status Code: {0}".format(r.status_code))
             logging.info("Successfully authenticated to {0}".format(self.ticketing_tool))
+            return s
```

Comment:
self.s is being set in __init__() on line 45. This allows me to raise a TicketException if a session was not successfully created in _create_requests_session(). Does this make sense? Or do you think there's a better way to do it? Any suggestions are welcome. :)